.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [3.0, 3.1, 3.2]
+        ruby: ['3.0', 3.1, 3.2]
```

Comment:
__comment:__ This fixes a [known issue](https://bibwild.wordpress.com/2021/12/28/github-action-setup-ruby-needs-to-quote-3-0-or-will-end-up-with-ruby-3-1/) with GitHub Actions.